src/React.AspNet.Middleware/AspNetFileSystem.cs

```diff
@@ -36,12 +36,12 @@ public AspNetFileSystem(IHostingEnvironment hostingEnv)
 		/// <returns>Full path of the file</returns>
 		public override string MapPath(string relativePath)
 		{
-            if (relativePath.StartsWith(_hostingEnv.WebRootPath))
-            {
-                return relativePath;
-            }
-            relativePath = relativePath.TrimStart('~').TrimStart('/');
-            return Path.Combine(_hostingEnv.WebRootPath, relativePath);
+			if (relativePath.StartsWith(_hostingEnv.WebRootPath))
+			{
+				return relativePath;
+			}
+			relativePath = relativePath.TrimStart('~').TrimStart('/').Replace("/", "\\");
```

Comment:
Shouldn't `Path.Combine` be normalizing the path already?

Reply:
Unfortunately it doesn't fix slashes :( :(

Reply:
Oh! Path.GetFullPath does convert slashes, so that's a more elegant fix.